android/build.sbt

```diff
@@ -0,0 +1,30 @@
+BaseProject.settings
+
+import android.Keys._
+
+android.Plugin.androidBuildAar
+
+platformTarget in Android := "android-22"
+
+name := "nordpol-android"
+
+// Do not append Scala versions to the generated artifacts
+crossPaths := false
+
+// This forbids including Scala related libraries into the dependency
```

Comment:
I think this comment should rather be: "Prevents the scala stdlib from beeing included automatically."